se2_planning/src/OmplReedsSheppPlanner.cpp

```diff
@@ -63,6 +69,35 @@ void OmplReedsSheppPlanner::setStateSpaceBoundaries() {
   bounds_->high[0] = parameters_.xUpperBound_;
   bounds_->high[1] = parameters_.yUpperBound_;
   stateSpace_->as<ompl::base::SE2StateSpace>()->setBounds(*bounds_);
+  //  std::cout << "OmplReedsSheppPlanner: Set state space bounds: x = [" << bounds_->low[0] << ", " << bounds_->high[0] << "], y=["
+  //            << bounds_->low[1] << ", " << bounds_->high[1] << "]" << std::endl;
+}
+const ompl::base::RealVectorBounds& OmplReedsSheppPlanner::getStateSpaceBoundaries() {
+  return stateSpace_->as<ompl::base::SE2StateSpace>()->getBounds();
+}
+
+void OmplReedsSheppPlanner::updateStateSpaceBounds(const ompl::base::RealVectorBounds& bounds) {
+  bounds_->low[0] = bounds.low[0];
+  bounds_->low[1] = bounds.low[1];
+  bounds_->high[0] = bounds.high[0];
+  bounds_->high[1] = bounds.high[1];
```

Comment:
you can make it virtual and provide a default implementation in the base class that updates bounds_, the parameters can be updated in a method inside the derived class.

Reply:
Done in db4a618de5f69d9b9b0bee5375f1217ca36002a6.